community/sharing.md

```diff
@@ -0,0 +1,51 @@
+# Sharing your Mods
+Author(s): code3z (@code3 on the forum)
+
+Once you've created a mod that you're proud of, the next step is to share it. Sharing will be a big help to other people. You can also get feedback and help from the community. But it may also mean you have to respond to bugs in your code and help other people use your code.
+
+
+## Code headers
+
+Putting a comment at the top of your code helps people understand it, use it, and remember where it came from. Consider using this template:
+```
+/**
+ * <TITLE>
+	- - -   
+ * Description:        <description>
+ * Filename:          <filename>
+ * Platform:          <Linux/Mac/Windows/All>
+ * Version:           Vivaldi version <version>
+ * Author(s):          <github username, forum username, other contributors>
+ * Topic:             https://forum.vivaldi.net/topic/<topic>
+**/
+```
+
+### Versioning Your Mods
+
+Instead of tracking each edit to your mod as a separate version, its easier and sometimes more helpful to include the latest version of Vivaldi that it was tested in. Along with the last update time, this will let you and others keep track of the mod version.
+
+Because the forum and Git keep track of posting times, it is probably not necessary to include the last update time in the code header (but you can if you want).
+
+If you do want to version your mods, it would look like this:
+```
+Version: 2.1 for Vivaldi 3.9.2289.3
+```
+
+If you want to keep it simple and use the last vivaldi version, it would look like this (see the header template above):
+```
+* Version:           Vivaldi 3.9.2289.3
+```
+
+## Tagging your Mods
```

Comment:
Since the rest of the file is about sharing in general, could you make it more clear that you mean tagging forum topics?